platform/ios/test/MGLMapViewPitchTests.m

```diff
@@ -0,0 +1,156 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+static MGLMapView *mapView;
+static CGFloat mbx_tiltGestureYTranslation;
+static NSUInteger mbx_numberOfFingersForGesture;
+
+@interface MGLMapView (MGLMapViewPitchTests)
+- (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)rotate;
+- (MGLMapCamera *)cameraByTiltingToPitch:(CGFloat)pitch;
+@end
+
+@interface UIPanGestureRecognizer (MGLMapViewPitchTests)
+@property (nonatomic, readwrite) UIGestureRecognizerState state;
+@end
+
+@implementation UIPanGestureRecognizer (MGLMapViewPitchTests)
+- (NSUInteger)numberOfTouches { return mbx_numberOfFingersForGesture; }
```

Comment:
Is this guaranteed to work?

Reply:
Oh, this is a subclass, so should be ok?

Reply:
Well... it works, but I’m not sure about “guaranteed” and certainly this sort of thing is discouraged in most situations. Moved to a subclass in https://github.com/mapbox/mapbox-gl-native/pull/15028/commits/322dce36db797098e4ae5e9b540a9a54bbddc1e1.